src/main/java/alfred/task/Todo.java

```diff
@@ -0,0 +1,20 @@
+package alfred.task;
+
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    public String getType() {
+        return "T";
+    }
+
+    public String getDate() {
+        return "";
```

Comment:
If the Todo class in this context has no "date" as in the Deadline or Event, why would this class have a method getDate that returns ""? Maybe the getDate method in the parent class (Task) does not have to be an abstract method?

Reply:
Hi Mick, the rationale behind having an abstract `getDate()` method was so that I am able to call `Task.getDate()` when encoding tasks to be saved in my `.txt` file, and do not have to write separate methods for encoding different types of tasks like `Todo`s, `Event`s, and `Deadline`s. May I check if this is acceptable? If not, I would have to create more methods to handle tasks with dates separately.

Reply:
I understand. I think your response makes sense to me. Thank you for the prompt response.